networkoperator/v1/001-egressrouter-crd.yaml

```diff
@@ -0,0 +1,231 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.0
```

Comment:
this line can go, can't it? Or will controller-gen readd it?

Reply:
I can try to make it go away, and see what happens.